src/commands/decrypt.js

```diff
@@ -63,7 +63,10 @@ const decrypt = async (withPassword, sessionKeyOut, withSessionKey, verifyWith,
     decryptionKeys = await Promise.all(decryptionKeys.map(privateKey => openpgp.decryptKey({
       privateKey,
       passphrase: [keyPassword, keyPassword.trimEnd()]
-    })));
+    }))).catch((e) => {
```

Comment:
maybe let's add a comment here that this is not exactly following the spec?

Reply:
Good idea, done :)